README.md

```diff
@@ -1,173 +1,126 @@
 
-+ Signaling Server (Node.JS) - https://github.com/anastasiia-zolochevska/signaling-server
-+ TURN Server (optional, needed for VPN/Proxy networks) - https://github.com/anastasiia-zolochevska/coturn-to-azure-deployment
+Here's a table illustrating where each sample implementation will be built to. To run one, navigate to that location and start the `exe`.
 
-## Building WebRTC Libraries from Source (You don't need to do this unless you are changing the encoder)
+> Note: the following table describes the location under `Build\<Platform>\<Configuration>\` where a sample can be found. To identify what `<Platform>` and `<Configuration>` are, see your desired configuration from [section: the actual build](#the-actual-build).
 
-> Note: If you want to build webrtc yourself, you will need to install Visual Studio 2015 with Update 3.
+<table>
+<tr>
+<th>Sample Type</th>
+<th>Sample Name</th>
+<th>Location</th>
+<th>Exe</th>
+</tr>
+<tr>
+<td>Server</td>
+<td>Spinning Cube</td>
+<td>SpinningCubeServer</td>
+<td>`SpinningCubeServer.exe`</td>
+</tr>
+<tr>
+<td>Server</td>
+<td>MultiThreaded</td>
+<td>MultiThreadedServer</td>
+<td>`MultiThreadedServer.exe`</td>
+</tr>
+<tr>
+<td>Client</td>
+<td>Win32Client</td>
+<td>Client</td>
+<td>`StreamingDirectXClient.exe`</td>
+</tr>
+</table>
 
-[Visual Studio 2015 Update 3 Community Edition ISO](http://download.microsoft.com/download/b/e/d/bedddfc4-55f4-4748-90a8-ffe38a40e89f/vs2015.3.com_enu.iso)
+Once you start both a server and client implementation, you should be seeing success! If you're instead seeing errors, check out the [Troubleshooting guide](#todo-link) and then [file an issue](https://github.com/catalystcode/3dtoolkit/issues/new). Additionally, you can see more information about our other sample implementations [here](#todo-link).
 
-[Visual Studio 2015 Update 3 Professional Edition ISO](http://download.microsoft.com/download/e/b/c/ebc2c43f-3821-4a0b-82b1-d05368af1604/vs2015.3.pro_enu.iso)
+## Specifics
 
-[Visual Studio 2015 Update 3 Enterprise Edition ISO](http://download.microsoft.com/download/8/4/3/843ec655-1b67-46c3-a7a4-10a1159cfa84/vs2015.3.ent_enu.iso)
+These resources will be critical to your success in configuring and scaling applications.
 
-Be sure to install all of the C++ language tools and the Windows Universal Components.
++ [Sample implementation configuration files](#todo-link)
++ [Configuring and deploying services](#todo-link)
++ [Configuring authentication](#todo-link)
 
-Once finished installing, install the [Windows DDK version 1607](https://go.microsoft.com/fwlink/p/?LinkId=526733) as it contains debugging tools needed by WebRTC.
+## Learn More
 
-Finally, launch PowerShell with Administrative persmissions and run 
-```
-.\WebRTCLibs\webrtcSetup.ps1 -WebRTCFolder C:\path\here
-```
-(If path is omitted, it will install to `C:\WebRTCSource`. Be careful with long path names, depot tools may fail to install.)
++ [General Wiki]()
```

Comment:
these should be `#todo-link` until they're actually linked. (well really, the next pass should be actually linking them)